src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,54 +12,59 @@ export const TodoFilter: React.FC<Props> = ({
   onSelectChange,
   query,
   select,
-}) => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select
-          data-cy="statusSelect"
-          value={select}
-          onChange={(e) => onSelectChange(e.target.value)}
-        >
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+}) => {
+  const onSelectHandler = (e: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
Follow [these](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) naming conventions for methods